Formula/irssi.rb

```diff
@@ -23,12 +43,14 @@ class Irssi < Formula
 
   head do
     url "https://github.com/irssi/irssi.git"
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
```

Comment:
Let's keep these. They don't hurt, and I worry someone will forget to put them back once the patch is removed.

Reply:
Good call, done.